src/entities/models/dao.py

```diff
@@ -37,11 +36,33 @@ class FilingPeriodDAO(Base):
     filing_type: Mapped[FilingType] = mapped_column(SAEnum(FilingType))
 
 
+class FilingTaskDAO(Base):
+    __tablename__ = "filing_task"
+    name: Mapped[str] = mapped_column(primary_key=True)
+    task_order: Mapped[int]
+
+    def __str__(self):
+        return f"Name: {self.name}, Order: {self.task_order}"
+
+
+class FilingTaskStateDAO(Base):
+    __tablename__ = "filing_task_state"
+    filing: Mapped[int] = mapped_column(ForeignKey("filing.id"), primary_key=True)
+    task_name: Mapped[str] = mapped_column(ForeignKey("filing_task.name"), primary_key=True)
+    task: Mapped[FilingTaskDAO] = relationship(lazy="selectin")
+    user: Mapped[str]
+    state: Mapped[FilingTaskState] = mapped_column(SAEnum(FilingTaskState))
+    change_timestamp: Mapped[datetime] = mapped_column(server_default=func.now(), onupdate=func.now())
+
+    def __str__(self):
+        return f"Filing ID: {self.filing}, Task: {self.task}, User: {self.user}, state: {self.state}, Timestamp: {self.change_timestamp}"
+
+
 class FilingDAO(Base):
     __tablename__ = "filing"
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     lei: Mapped[str]
-    state: Mapped[FilingState] = mapped_column(SAEnum(FilingState))
+    tasks: Mapped[List[FilingTaskStateDAO]] = relationship(lazy="selectin", cascade="all, delete-orphan")
     filing_period: Mapped[int] = mapped_column(ForeignKey("filing_period.id"))
     institution_snapshot_id: Mapped[str]
     contact_info: Mapped[str] = mapped_column(nullable=True)
```

Comment:
good for this PR, but let's create a ticket to get contact_info as a separate table

Reply:
https://github.com/cfpb/sbl-filing-api/issues/62